### PR TITLE
prototype of new README.md file for s2i repositories

### PR DESCRIPTION
**Pull request by ficap**
according to new rules
@hhorak please review new stuff

### CODE REVIEW

**Comment by ficap** [comment]
and how about troubleshooting article. does it also logs to term so it appears in docker logs or developer of the application can have his/her own way to log the application for example logging to special file stored somewhere based on developer.

File: 3.5/README.md [hunk 1]
```diff
@@ -244,3 +255,11 @@ docker exec -it <CONTAINER_ID> /bin/bash
 
 After you enter into the running container, your current directory is set
 to `/opt/app-root/src`, where the source code is located.
+
+
+See also
+--------
+Base Python environment Dockerfile and other sources for building the final image 
+with your Python application are available on https://github.com/sclorg/s2i-python-container.
```
Comment (ficap) [inline 3.5/README.md]: containing your Python application - 'd be better?

Comment (pkubatrh) [inline 3.5/README.md]: Neither. You do not need to have the sources for s2i-python to be able to build an image containing your python application. Source-to-image does all the magic for you.
The plain "Dockerfile and other sources for this container image are available on" would be better.

**Comment by torsava** [comment]
Looks good to me.

**Comment by pkubatrh** [comment]
The description seems like it has been directly taken from Python's documentation. As I am not familiar with Python's license, are we allowed to do so without attribution?

In general I think it would be better to just use something we already have, like what is available in Fedora rpms:
https://admin.fedoraproject.org/pkgdb/package/rpms/python/

**Comment by ficap** [comment]
I used description from Dockerfile DESC, so if there's any licence problem there is also.

**Comment by pkubatrh** [comment]
>I used description from Dockerfile DESC, so if there's any licence problem there is also.

It might be just me over-thinking things in that case. Feel free to add changes to other versions as well.

**Comment by ficap** [comment]
Should I squash these commits into one?

**Comment by omron93** [comment]
@ficap Github allows squashing of PR when merging. But you can.

```
 Squash and merge
The 3 commits from this branch will be combined into one commit in the base branch. 
```

**Comment by ficap** [comment]
@hhorak please review latest changes

**Comment by hhorak** [comment]
@ficap the 3.5 version looks good to me, but the last changes are not done in 2.7 and 3.4 versions. The PR also will need to be rebased.

**Comment by hhorak** [comment]
Otherwise no issues spotted.

**Comment by hhorak** [comment]
This looks good to me, let's merge it and thanks!

**Pull request merged**